token/js/src/extensions/transferFee/instructions.ts

```diff
@@ -826,3 +826,158 @@ export function decodeHarvestWithheldTokensToMintInstructionUnchecked({
         },
     };
 }
+
+// SetTransferFee
+
+export interface SetTransferFeeInstructionData {
+    instruction: TokenInstruction.TransferFeeExtension;
+    transferFeeInstruction: TransferFeeInstruction.SetTransferFee;
+    transferFeeBasisPoints: number;
+    maximumFee: bigint;
+}
+
+export const setTransferFeeInstructionData = struct<SetTransferFeeInstructionData>([
+    u8('instruction'),
+    u8('transferFeeInstruction'),
+    u16('transferFeeBasisPoints'),
+    u64('maximumFee'),
+]);
+
+/**
+ * Construct a SetTransferFeeInstruction instruction
+ *
+ * @param mint                      The token mint
+ * @param authority                 The source account's owner/delegate
```

Comment:
Sorry! I missed this last pass.

```suggestion
 * @param authority                 The authority of the transfer fee
```